### PR TITLE
[Hold] Update attack-simulator.md

### PR DESCRIPTION
**Pull request by garber-001**
Fixed direct attach simulator link

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/attack-simulator.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ If your organization has Microsoft Defender for Office 365 Plan 2, which include
 
 ## What do you need to know before you begin?
 
-- To open the Security & Compliance Center, go to <https://protection.office.com/>. Attack simulator is available at **Threat management** \> **Attack simulator**. Go go directly to attack simulator, open <https://protection.office.com/attacksimulator>.
+- To open the Security & Compliance Center, go to <https://protection.office.com/>. Attack simulator is available at **Threat management** \> **Attack simulator**. Go go directly to attack simulator, open <https://security.microsoft.com/attacksimulator>.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/attack-simulator.md]:
```suggestion
- To open the Security & Compliance Center, go to <https://protection.office.com/>. Attack simulator is available at **Threat management** \> **Attack simulator**. To go directly to Attack simulator, open <https://security.microsoft.com/attacksimulator>.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by chrisda** [comment]
@garber-001, there are a couple of reasons why I'm thinking about not approving this:

- This is the original Attack simulator that is now read only, will basically go away in 90 days, and has been replaced by Attack simulation training.
- This old version of Attack simulator is 100% available in the Security & Compliance Center. At some point in the near future, ALL existing procedures (including ancillary links like permission statements) will be updated/migrated from the Security & Compliance Center to the Microsoft 365 compliance  center.

Comments?

**Comment by chrisda** [comment]
Closing

**Pull request closed without merging**